examples/cluster/Dockerfile.mesos

```diff
@@ -12,17 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:trusty
+FROM mesosphere/mesos:1.2.0
```

Comment:
Rather than building this image, can we just specify mesosphere/mesos:1.2.0 in docker-compose file? We can mount mesos-secrets and mesos-slave-secrets to the container using volumes option in docker-compose.